ml/rl/workflow/sample_configs/cartpole_discrete_dqn_online.yaml

```diff
@@ -0,0 +1,67 @@
+%YAML 1.2
+---
+env: CartPole-v0
+model:
+  DiscreteDQN:
+    trainer_param:
+      actions:
+      - 4
+      - 5
+      rl:
+        gamma: 0.99
+        epsilon: 0.05
+        target_update_rate: 0.1
+        maxq_learning: true
+        reward_boost:
+        temperature: 0.35
+        softmax_policy: 0
+        use_seq_num_diff_as_time_diff: false
+        q_network_loss: mse
+        set_missing_value_to_zero: false
+        tensorboard_logging_freq: 0
+        predictor_atol_check: 0
+        predictor_rtol_check: 5.0e-05
+        time_diff_unit_length: 1
+        multi_steps:
+        ratio_different_predictions_tolerance: 0
+      double_q_learning: true
+      bcq:
+      minibatch_size: 512
+      minibatches_per_step: 1
+      optimizer:
+        optimizer: ADAM
+        learning_rate: 0.01
+        l2_decay: 0
+      evaluation:
+        calc_cpe_in_training: false  # true
+    net_builder:
+      FullyConnected:
+        sizes:
+        - 128
+        - 64
+        activations:
+        - relu
+        - relu
+    cpe_net_builder:
+      FullyConnected:
+        sizes:
+        - 256
+        - 128
+        activations:
+        - relu
+        - relu
+        dropout_ratio: 0
+    target_action_distribution:
+    preprocessing_options:
+      feature_overrides:
+      tablesample:
+      whitelist_features:
+      max_unique_enum_values: 0
+    reader_options:
```

Comment:
Could you remove the config lines that are either default or null? This looks a bit overwhelming.